orc8r/cloud/go/services/analytics/analytics/main.go

```diff
@@ -25,7 +25,7 @@ import (
 )
 
 const (
-	defaultAnalysisSchedule = "*/1 * * * *" // Every 1 minute
+	defaultAnalysisSchedule = "*/15 * * * *" // Every 15 minutes
 )
 
 //ServiceMain - entrypoint for analytics service
```

Comment:
This comment is superfluous

Reply:
Not everyone might be aware of cron format. I'm planning on cleaning this up in follow up review.

Reply:
Sorry to clarify: the "ServiceMain" comment is the one that's superfluous. I think adding the cron comment is appropriate